### PR TITLE
Add missing env variables to maven container

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,24 +1,23 @@
 def mavenVersion='3.3.9'
+node {
```
Comment (rhuss) [inline Jenkinsfile]: Why is this added ?

File: Jenkinsfile [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
+    slave {
+        withOpenshift {
+                withMaven(mavenImage: "maven:${mavenVersion}", 
+		envVars: [ containerEnvVar(key:'GITHUB_OAUTH_CLIENT_ID', value: "${env.GITHUB_OAUTH_CLIENT_ID}"), containerEnvVar(key:'GITHUB_OAUTH_CLIENT_SECRET', value: "${env.GITHUB_OAUTH_CLIENT_SECRET}") ],
```
Comment (rhuss) [inline Jenkinsfile]: The env vars evaluated by whom ?

File: Jenkinsfile (hunk 1 above)
Comment (iocanel) [inline Jenkinsfile]: This is added so that we can have access to env.

File: Jenkinsfile (hunk 2 above)
Comment (iocanel) [inline Jenkinsfile]: The env vars are evaluated by the jenkins master. The result is passed to maven container so that they can be used to setup the test environment.

**Comment by rhuss** [comment]
Thanks, let's merge
